lib/libc/netdb/lib_getaddrinfo.c

```diff
@@ -51,6 +51,7 @@
 
 #include <netdb.h>
 #include <errno.h>
+#include <arpa/inet.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?